src/blocks/Slider/Slider.tsx

```diff
@@ -289,14 +307,20 @@ export const SliderBlock = (props: React.PropsWithChildren<SliderProps>) => {
     };
 
     const renderDot = (index: number) => {
+        const isVisible = isVisibleSlide(index);
         return (
             <li
                 key={index}
                 className={b('dot', {active: index === currentIndex})}
                 onClick={() => handleDotClick(index)}
-                aria-hidden={isVisibleSlide(index) ? true : undefined}
-                aria-label={`Slide ${getCurrentSlideNumber(index)} of ${barSlidesCount}`}
-            ></li>
+                role="button"
```

Comment:
`li` should have only `listitem` or `generic` role. It is not recommended to change it: https://www.w3.org/TR/html-aria/#el-li